pkg/oc/lib/graph/genericgraph/test/prereq-image-present-notag.yaml

```diff
@@ -55,7 +55,7 @@ items:
     strategy:
       dockerStrategy:
         from:
-          kind: DockerReference
+          kind: DockerImage
```

Comment:
@bparees that was weird change, do we somehow convert `DockerReference` to `DockerImage` during conversion from external to internal, I coulnd't find anything like that. If not, how this could ever work?

Reply:
I can only speculate that this object was never being passed through build api validation?

because i don't think "DockerReference" would have ever been correct.

Reply:
Yeah, that what I imagined as well, that's why this is now fixed :)

Reply:
@soltysh would be nice if we can call the validation in test to ensure the objects are actually valid :-) (follow up issue?)